sample/src/androidTest/java/com/schibsted/spain/barista/sample/SwipeRefreshTest.java

```diff
@@ -0,0 +1,22 @@
+package com.schibsted.spain.barista.sample;
+
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static com.schibsted.spain.barista.BaristaAssertions.assertDisplayed;
+import static com.schibsted.spain.barista.BaristaSwipeRefreshActions.pullToRefresh;
+
+@RunWith(AndroidJUnit4.class) public class SwipeRefreshTest {
+
+  @Rule public ActivityTestRule<SwipeRefreshActivity> activityRule =
+      new ActivityTestRule<>(SwipeRefreshActivity.class);
+
+  @Test
+  public void testSwipeRefresh_isRefreshing() {
+    pullToRefresh(R.id.swiperefresh);
+    assertDisplayed("I am refreshing!");
```

Comment:
Thanks for this test! I know that writing the test and the Activity that mimics an actual application is a boring task... thanks! 👍